File: doc/codes/python/examples/api/compute/pattern/uniform-example.py
from autd3.geometry import Autd3, Geometry
from autd3.value import Emission, Intensity, Phase
from autd3_pattern import uniform

geometry = Geometry([Autd3([0.0, 0.0, 0.0], [1.0, 0.0, 0.0, 0.0])])

out = geometry.pattern_buffer()

uniform(
    Emission(
        phase=Phase.ZERO,
        intensity=Intensity.MAX,
    ),
    out,
)

File: doc/codes/rust/examples/api/compute/pattern/uniform-example.rs
use autd3_rs::geometry::{Autd3, Geometry};
use autd3_rs::value::{Emission, Intensity, Phase};
use autd3_rs_pattern::uniform;

// HIDE
fn main() {
    // HIDE_END
    let geometry = Geometry::new(vec![Autd3::default()]);

    let mut dst = geometry.pattern_buffer();

    uniform(
        Emission {
            phase: Phase::ZERO,
            intensity: Intensity::MAX,
        },
        &mut dst,
    );
    // HIDE
}
// HIDE_END
